### PR TITLE
SettingsContract.java file converted to kotlin #1826

### PR DESCRIPTION
**Pull request by Rishavgupta12345**
SettingsContract.java file converted to kotlin  #1826

### CODE REVIEW

File: core/src/main/java/org/kiwix/kiwixmobile/core/settings/SettingsContract.kt [hunk 1]
```diff
@@ -15,17 +15,13 @@
...
-import org.kiwix.kiwixmobile.core.base.BaseContract;
-
-interface SettingsContract {
-  interface View extends BaseContract.View<Presenter> {
-
-  }
-
-  interface Presenter extends BaseContract.Presenter<View> {
-    void clearHistory();
+internal interface SettingsContract {
```
Comment (gouri-panda) [inline core/src/main/java/org/kiwix/kiwixmobile/core/settings/SettingsContract.kt:22]: Why do you think `internal` will be appropriate?

Comment (Rishavgupta12345) [reply core/src/main/java/org/kiwix/kiwixmobile/core/settings/SettingsContract.kt:22]: SHOULD I REMOVE THE INTERNAL ???

Comment (gouri-panda) [reply core/src/main/java/org/kiwix/kiwixmobile/core/settings/SettingsContract.kt:22]: Ok and also rebase your commits into one.

Comment (Rishavgupta12345) [reply core/src/main/java/org/kiwix/kiwixmobile/core/settings/SettingsContract.kt:22]: sir my this branch has been corrupted with some unwanted commits and stuffs can i delete this pull request and create a fresh one ???

Comment (gouri-panda) [reply core/src/main/java/org/kiwix/kiwixmobile/core/settings/SettingsContract.kt:22]: You can close this and open a new one.

**Pull request closed without merging**